checkov/bitbucket_pipelines/runner.py

```diff
@@ -0,0 +1,135 @@
+import os
+
+from checkov.bitbucket_pipelines.checks.registry import registry
+from checkov.common.images.image_referencer import ImageReferencer, Image
+from checkov.common.output.report import CheckType
+from checkov.yaml_doc.runner import Runner as YamlRunner
+
+
+class Runner(YamlRunner, ImageReferencer):
+    check_type = CheckType.BITBUCKET_PIPELINES
+
+    def __init__(self):
+        super().__init__()
```

Comment:
I don't think you need that, when you don't do anything extra inside `__init__`, it will call the parent method anyway